src/UIKit/UIGestureRecognizer.cs

```diff
@@ -319,4 +319,27 @@ public void Activated (UIScreenEdgePanGestureRecognizer sender)
 #endif
 }
 
+// start code for UIHoverGestureRecognizer Constructor
+	public partial class UIHoverGestureRecognizer : UIGestureRecognizer {
+		public UIHoverGestureRecognizer (Action action) : base (action) {}
+		public UIHoverGestureRecognizer (Action<UIHoverGestureRecognizer> action) : base (Selector.GetHandle (UIGestureRecognizer.parametrized_selector), new Callback (action)) {}
+
+		[Register ("__UIHoverGestureRecognizer")]
+		class Callback : Token {
+			Action<UIHoverGestureRecognizer> action;
+			
+			internal Callback (Action<UIHoverGestureRecognizer> action)
+			{
+				this.action = action;
+			}
+			
+			[Export ("target:")]
+			[Preserve (Conditional = true)]
+			public void Activated (UIHoverGestureRecognizer sender)
+			{
+				action (sender);
+			}
```

Comment:
Little trick in the new c# syntax, you can do:
```suggestion
			public void Activated (UIHoverGestureRecognizer sender) => action (sender);
```